<launch>
    <node name="hsv_filter_node" pkg="hsv_filter" type="hsv_filter_node" output="screen">
        <param name="h_low" value="0" type="int" />
        <param name="s_low" value="0" type="int" />
        <param name="v_low" value="0" type="int" />
        <param name="h_high" value="220" type="int" />
        <param name="s_high" value="230" type="int" />
        <param name="v_high" value= "200" type="int" />
        <remap from="/camera/image_raw" to="uwsim/camera2"/>
    </node>
</launch>
